src/components/widgets/gcode-preview/GcodePreviewCard.vue

```diff
@@ -17,6 +17,18 @@
         </app-btn>
       </app-btn-collapse-group>
 
+      <app-btn
+        color=""
+        class="ml-1"
+        fab
+        x-small
+        text
+        :disabled="!parts.length"
```

Comment:
Start a print, wait for this to enable, press CANCEL... still enabled!
![image](https://user-images.githubusercontent.com/85504/178109007-a1d2d751-1eea-431c-be38-f383d65820aa.png)
```suggestion
        :disabled="!klippyReady || !printerPrinting || !printerFileLoaded || !parts.length"
```